cypress_test/integration_tests/desktop/calc/help_dialog_update_spec.js

```diff
@@ -44,4 +45,30 @@ describe(['tagscreenshot'], 'Help dialog update', function() {
 		});
 	});
 
+	it('JSDialog popup dialog', function() {
```

Comment:
nice test! but it seems to be unrelated to other tests in this file...
Can we put that into separate file? like   `jsdialog_spec.js`    so we will have there jsdialog specific tests

Reply:
I could not find a proper to file to add the unit test but "jsdilalog_spec.js" is Ok, I guess it is a matter to add the file and extra config file

Reply:
@hcvcastro I think for cypress it is enough to add new file in the directory, it will be automatically executed during check